docs/data/joy/components/typography/TypographyBasics.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import Card from '@mui/joy/Card';
+import Typography from '@mui/joy/Typography';
+
+export default function TypographyBasics() {
```

Comment:
This is a great demo to introduce this component! I would just tweak the styles a bit to give the text a bit more visual hierarchy. Something like this maybe: https://codesandbox.io/s/priceless-dijkstra-vdzq58?file=/demo.tsx